src/apps/opening-hours/OpeningHoursHelpers.ts

```diff
@@ -0,0 +1,80 @@
+import dayjs from "dayjs";
+import "dayjs/locale/da";
+import weekOfYear from "dayjs/plugin/weekOfYear";
+import { DplOpeningHoursListGET200Item as OpeningHoursItem } from "../../core/dpl-cms/model";
+
+dayjs.extend(weekOfYear);
+export const getNextWeek = (date: Date): Date => {
+  return dayjs(date).add(1, "week").toDate();
+};
+
+export const getPreviousWeek = (date: Date): Date => {
+  return dayjs(date).subtract(1, "week").toDate();
+};
+
+export const getWeekDisplay = (date: Date): string => {
+  const week = dayjs(date).week();
+  const year = dayjs(date).year();
+
+  return `Uge ${week}, ${year}`;
+};
+
+const capitalizeString = (string: string): string => {
+  return string.charAt(0).toUpperCase() + string.slice(1);
+};
+
+export const formatDateToWeekday = (date: Date): string => {
+  const formattedAsWeekday = dayjs(date).locale("da").format("dddd");
```

Comment:
Consider replacing all the `.locale("da")` with [setting a global locale](https://day.js.org/docs/en/i18n/changing-locale#docsNav).